RESideMenu/RESideMenuController.m

```diff
@@ -544,27 +523,26 @@ -(void)showMenuControllerFromDirection:(RESideMenuControllerDirection)menuDirect
     
     // Perspective rotation animation
     CABasicAnimation *animation = [CABasicAnimation animationWithKeyPath:@"transform.rotation.y"];
-    animation.fromValue = @0.0;
+
+    animation.fromValue = [(NSNumber *)self.perspectiveAnimationLayer valueForKeyPath:@"transform.rotation.y"];
```

Comment:
Is this cast in the right place? Looks like you're casting the layer to `NSNumber`?